streams/shared/src/main/scala/scalaz/stream/ZStream.scala

```diff
@@ -660,6 +660,16 @@ trait ZStream[-R, +E, +A] extends Serializable { self =>
         }
       }
   }
+
+  /**
+   * @note in case you need extreme performance, make sure to use capacity
```

Comment:
Good note, although let's phrase it as the common case: "when possible, prefer capacities that are powers of 2".

Reply:
Done